packages/react/src/Overlay/Overlay.docs.json

```diff
@@ -143,6 +143,13 @@
     {
       "name": "sx",
       "type": "SystemStyleObject"
+    },
+    {
+      "name": "data-responsive",
```

Comment:
maybe call this responsive variant instead?

Reply:
I don't feel strongly either way, I don't see why not make it public though 👀